src/comps/Deposit.tsx

```diff
@@ -226,30 +226,32 @@ const DepositFlowConfirm = ({
   const walletInfo = useAtomValue(walletInfoAtom);
   const handleNextClick = async () => {
     try {
+      const signersAggregatePubKey = (await getAggregateKey()).slice(2);
+
       // Combine the version and hash into a single Uint8Array
       const serializedAddress = serializeCVBytes(principalCV(stxAddress));
 
       // Parse lockTime from env variable
       const parsedLockTime = parseInt(lockTime || "144");
       // Create the reclaim script and convert to Buffer
       const reclaimScript = Buffer.from(
-        createReclaimScript(parsedLockTime, new Uint8Array([]))
+        createReclaimScript(parsedLockTime, new Uint8Array([])),
```

Comment:
nit: there's a lot of noise on PRs for this, are our tools using different configs?